src/lib/components/backoffice/ChangedBy/ChangedBy.js

```diff
@@ -14,10 +14,14 @@ const ChangedBy = ({ metadata, field }) => {
   } else {
     switch (byType) {
       case 'user_id':
-        return (
+        return byValue != -2 ? (
```

Comment:
that is right, but also we might hit the js types problem... because the patron_pid is string not a number (for sure in loans), so be careful if it is working with comparison operators

Reply:
According to this https://javascript.info/comparison#comparison-of-different-types there should be no problem with the types